plugin/set.go

```diff
@@ -106,11 +116,39 @@ func (i *Set) Run() error {
 	return i.RunCommand(args...)
 }
 
+func (i *Set) useProtobuf(args ...string) []string {
```

Comment:
Just for our sake of our future self lets just document that this will fail for older plugins and plugins that don't support this flag.